src/extension/flutter/flutter_daemon.ts

```diff
@@ -47,15 +48,20 @@ export class FlutterDaemon extends StdIOService<UnknownNotification> implements
 		if (process.env.DART_CODE_IS_TEST_RUN)
 			daemonArgs.push("--show-test-device");
 
-		const execution = usingCustomScript(
-			path.join(workspaceContext.sdks.flutter, flutterPath),
-			["daemon"].concat(daemonArgs),
-			workspaceContext.config?.flutterDaemonScript,
-		);
 
-		const flutterAdditionalArgs = config.for(vs.Uri.file(folder)).flutterAdditionalArgs;
-		const args = getGlobalFlutterArgs().concat(flutterAdditionalArgs).concat(execution.args);
-		this.createProcess(folder, execution.executable, args, { toolEnv: getToolEnv() });
+		if (!!config.daemonPort) {
+			this.createNcProcess(config.daemonPort);
```

Comment:
As above, if the user may set this in their global User Settings, we might need to make this only run for internal projects (eg. based on the presence of the custom Flutter script?).